cmd/autotestgridgenerator/main.go

```diff
@@ -67,45 +68,33 @@ func main() {
 	if err := validateOptions(o); err != nil {
 		logrus.WithError(err).Fatal("Invalid arguments.")
 	}
-
-	sa := &secret.Agent{}
-	if err := sa.Start([]string{o.GitHubOptions.TokenPath}); err != nil {
-		logrus.WithError(err).Fatal("Failed to start secrets agent")
+	if err := o.PRCreationOptions.Finalize(); err != nil {
+		logrus.WithError(err).Fatal("failed to finalize PR creation options")
 	}
 
-	gc, err := o.GitHubOptions.GitHubClient(sa, false)
-	if err != nil {
-		logrus.WithError(err).Fatal("error getting GitHub client")
-	}
-
-	stdout := bumper.HideSecretsWriter{Delegate: os.Stdout, Censor: sa}
-	stderr := bumper.HideSecretsWriter{Delegate: os.Stderr, Censor: sa}
-	author := fmt.Sprintf("%s <%s>", o.GitName, o.GitEmail)
-
 	command := "/usr/bin/testgrid-config-generator"
 	arguments := []string{
 		"-testgrid-config", o.testgridConfigDir,
 		"-release-config", o.releaseConfigDir,
 		"-prow-jobs-dir", o.prowJobsDir,
 		"-allow-list", o.allowList,
 	}
-	committed, err := bumper.RunAndCommitIfNeeded(stdout, stderr, author, command, arguments, o.workingDir)
-	if err != nil {
-		logrus.WithError(err).Fatal("failed to run command and commit the changes")
-	}
 
-	if !committed {
-		logrus.Info("no new commits, exiting ...")
-		os.Exit(0)
-	}
+	fullCommand := fmt.Sprintf("%s %s", command, strings.Join(arguments, " "))
+	logrus.Infof("Running: %s", fullCommand)
+	logrus.WithField("cmd", command).
+		// The default formatting uses a space as separator, which is hard to read if an arg contains a space
+		WithField("args", fmt.Sprintf("['%s']", strings.Join(arguments, "', '"))).
```

Comment:
Ideally we'd just not have spaces in args - which one does? Or just add the list as the field, the default print will be fine

Reply:
Ok, I'll just add it as a list